benchmarks/user/README.md

```diff
@@ -221,6 +223,61 @@ or to keep the memory footprint under control:
 watch numastat -m
 ```
 
+## Benchmarks' output
+
+The GFlops/s and GPoints/s performance, Operational Intensity (OI) and 
+execution time are emitted to standard output at the end of each run. 
+Further, when running in bench mode, a `.json` file is produced
+(see `python benchmark.py bench --help` for more info) in a folder named
+`results` except if otherwise specified with the `-r` option specifying
+the results directory.
+
+So the isotropic acoustic wave forward Operator in a `512**3` grid, space order
```

Comment:
How is this example any useful? we have analogous examples in the section above, and this doesn't seem to add anything. We can probably drop this

Reply:
It states the need for DEVITO_LOGGING =DEBUG (?)

Reply:
DEBUG is now automatic when running benchmark.py since the last related PR: https://github.com/opesci/devito/blob/master/benchmarks/user/benchmark.py#L408